serving/resource_model_serving.go

```diff
@@ -29,6 +28,16 @@ func ResourceModelServing() *schema.Resource {
 				return old == "" && new == "CPU"
 			}
 			common.MustSchemaPath(m, "config", "traffic_config").Computed = true
+			common.MustSchemaPath(m, "config", "served_models").Deprecated = "Please use 'config.served_entities' instead of 'config.served_models'."
```

Comment:
We need to add `ConflictsWith` between `served_models` and `served_entities` otherwise people could try to specify both